### PR TITLE
Fix(CSS): `override` doesn't work

### PR DESCRIPTION
**Pull request by lesha1201**
Noticed in the code that config doesn't work with `override: true` because it passes `undefined` instead of `customConfig`. This PR makes it possible to use `override: true`.

### CODE REVIEW

File: packages/css/src/core.ts [hunk 1]
```diff
@@ -370,79 +370,98 @@ export class MasterCSS {
...
-                        }
-                        eachRuleOptions.resolvedMatch = new RegExp(`^${key}:(?:${valueMatches.join('|')})[^|]*?(?:@|$)`)
-                    } else {
-                        eachRuleOptions.resolvedMatch = match as RegExp
                     }
-                }
-            })
+                },
+            )
+        }
```
Comment (lesha1201) [inline packages/css/src/core.ts]: Here, I just added `if` statement and indented the code. I'm not sure how it actually should look like, I just did the same way as the code above.

File: packages/css/tests/config/override/test.ts
Comment (lesha1201) [inline packages/css/tests/config/override/test.ts:1]: Not sure if it's the right place for the tests.

**Comment by lesha1201** [comment]
@1aron How do you format the code? What do you think about adding Prettier?

**Comment by 1aron** [comment]
> @1aron How do you format the code? What do you think about adding Prettier?

We now use https://editorconfig.org. The reasons for not using Prettier: https://antfu.me/posts/why-not-prettier

File: packages/css/tests/config/override/test.ts
Comment (1aron) [reply packages/css/tests/config/override/test.ts:1]: Correct!

**Comment by lesha1201** [comment]
@1aron Formatted the code.

**Comment by 1aron** [comment]
@lesha1201 Released. Great job 👍🏻

**Pull request merged**